_posts/2022-01-07-Basics-of-Machine-Learning.md

```diff
@@ -0,0 +1,47 @@
+# The Basics of Machine Learning without math
+
+## Table of content
```

Comment:
Table of Content*s*

Reply:
Vielleicht gibt es auch eine Markdown-Erweiterung, mit der man das Inhaltsverzeichnis automatisch erstellen kann.
Ansonsten vielleicht auch sowas:
https://github.com/toshimaru/jekyll-toc

Dann sind die Zeilen auch verlinkt.